src/Model/Tag.php

```diff
@@ -432,53 +437,53 @@ private static function getTypeForHash(string $hash)
 	/**
 	 * Create implicit mentions for a given post
 	 *
-	 * @param integer $uri_id
-	 * @param integer $parent_uri_id
+	 * @param integer $uriId URI Id
+	 * @param integer $parentUriId Parent URI id
```

Comment:
Yes, I guessed those from the parameter names.
```suggestion
	 * @param integer $uriId
	 * @param integer $parentUriId
```